internal/cluster/connection_manager.go

```diff
@@ -60,6 +60,10 @@ const (
 	serializationVersion = 1
 )
 
+const (
+	clusterConnectionFixTimeout = 10 * time.Second
```

Comment:
what is this needed for ?  We already have connection timeout in NetworkConfig and also heartbeat timeout for this area? Why do we need another timeout ?

Reply:
That limits the connection time for the context created in `checkFixConnection`.That's to prevent a very slow connection to

Reply:
Removed that bit.